.github/workflows/backend.yml

```
@@ -73,15 +73,15 @@ jobs:
       - name: Prepare back-end environment
         uses: ./.github/actions/prepare-backend
         with:
-          m2-cache-key: 'kondo'
+          m2-cache-key: "kondo"
       - name: Run clj-kondo
         run: ./bin/kondo.sh
 
   be-linter-eastwood:
     needs: [files-changed, static-viz-files-changed]
     if: |
       !cancelled() &&
-      github.event.pull_request.draft == false &&
+      (github.event.pull_request.draft == false || (github.event.pull_request.draft == true && contains(github.event.pull_request.labels.*.name, 'run-tests'))) &&
```

Could we simplify this to "draft is false or "run-tests" label exists"?

```suggestion
      (github.event.pull_request.draft == false || (contains(github.event.pull_request.labels.*.name, 'run-tests')) &&
```

---

that's fair, it's definitely simpler. I don't like a lot of nested stuff either.